code/modules/mob/living/carbon/xenomorph/castes/defiler/abilities_defiler.dm

```diff
@@ -302,6 +302,10 @@
 /datum/action/xeno_action/activable/inject_egg_neurogas/use_ability(atom/A)
 	var/mob/living/carbon/xenomorph/defiler/X = owner
 
+	if(!owner.Adjacent(A))
+		A.balloon_alert(X, "Cannot reach")
```

Comment:
Minor wording feedback, but happy to merge regardless.
```suggestion
		A.balloon_alert(X, "Out of reach")
```

Reply:
I just used the same alert text from the defile ability